app/js/api/index.js

```diff
@@ -15,6 +15,7 @@ class APIRoot {
     this.Buy = new API("stock_transactions/buy");
     this.Sell = new API("stock_transactions/sell");
     this.Stock = new API("stocks");
+    this.Issues = new API("api/github_issues");
```

Comment:
So this probably only needs to remove ~~api~~; as you can see nothing else has that prefix.